fedimint-testing/src/fixtures.rs

```diff
@@ -187,7 +187,14 @@ impl Fixtures {
         let listen: SocketAddr = "127.0.0.1:9000".parse().unwrap();
         let address: SafeUrl = format!("http://{listen}").parse().unwrap();
 
-        let ln_client: Arc<dyn ILnRpcClient> = lightning_builder.build().await.into();
+        // TODO: Pass in the test runtime rather than spawning a new one.
+        let runtime = Arc::new(
+            tokio::runtime::Builder::new_multi_thread()
+                .enable_all()
+                .build()
+                .unwrap(),
+        );
```

Comment:
Ok looking at the 

```
fedimint-test-all-wasm32-unknown-unknown-ci> 00:00:37 thread 'lnv2_incoming_contract_with_invalid_preimage_is_refunded' panicked at /nix/store/zfdss5y3nhza1bdpkghb07i23f5q6lwg-vendor-cargo-deps/c19b7c6f923b580ac259164a89f2577984ad5ab09ee9d583b888f934adbbe8d0/tokio-1.41.0/src/runtime/blocking/shutdown.rs:51:21:
fedimint-test-all-wasm32-unknown-unknown-ci> 00:00:37 Cannot drop a runtime in a context where blocking is not allowed. This happens when a runtime is dropped from within an asynchronous context.
```

If we follow the context, this runtime is dropped when `new_gateway` is returning, so probably we should store the runtime somewhere. What do you think about storing it inside the `FakeLightningBuilder`? in this way when the `FakeLightningBuilder` drops, the runtime should drop too?

However, I still think that in the test the runtime should be pass down from the test to the fixtures, the question that it is hard to answer probably is how?